lib/blockscore/base.rb

```diff
@@ -92,6 +85,13 @@ def add_setter(symbol, *_args)
 
     private
 
+    def force!
+      res = @proc.call
+      @attributes = res.attributes.merge(@attributes)
+      @loaded = true
+      self
```

Comment:
Is this return value used anywhere? I can see where it could be used in the public interface, but the only caller I can see is calling the method in void context.